index.html

```diff
@@ -163,7 +163,7 @@ <h1>DRAFT</h1>
 						<td>
 							Dave Cramer (Hachette Livre),
 							Wendy Reid (Rakuten Kobo),
-							Shinya Takami (高見真也), (Kadokawa/Bookwalker)
+							Shinya Takami (高見真也) (Kadokawa/Bookwalker)
```

Comment:
We lost a comma here. I'm not sure of the most elegant way to format this information. In JLREQ Japanese names were in parenthesis, but affiliations were not. I can look for more examples. 

The diff here is quite large; there are a lot of white-space only changes. Not sure how we can avoid that.

Reply:
I'm not sure why when I opened it in Atom it changed all the whitespace :/.